### DISCUSSION TITLE
Disable dark mode?

### DISCUSSION THREAD

**Discussion by scottandrew** [skeletonlabs/skeleton#1117] [General]
I don't want dark mode is there a way to disable dark mode? Also what is the best way to just keep a white background and not a surface color?

**Comment by endigo9740** [comment]
@scottandrew Dark Mode is not a feature specific to Skeleton, it's just using Tailwind's underlying system for this. You can learn more about it here:
https://tailwindcss.com/docs/dark-mode

To answer your question, I recommend the following:

1. In your `tailwind.config.cjs` set `darkMode: "class"` per our installation recommendation. This won't enable dark mode, it simply edits the way it functions.
2. Open `/src/app.html` and ensure the `.dark` class is NOT applied to the `<html>` element at the top
3. Open `/src/app.postcss` and edit your HTML or Body element styles as you would with any other web app, such as:

```css
html { background: white; }
```
